Source/src/components/ComponentMeshRenderer.h

```diff
@@ -108,6 +108,16 @@ namespace Hachiko
             return tint_color;
         }
 
+        void SetDissolveProgress(const float progress)
+        {
+            dissolve_progress = progress;
+        }
+
+        [[nodiscard]] const float GetDissolveProgress() const
```

Comment:
You can remove the unnecessary const modifier for return type as the float is returned by value anyway.

Reply:
Oka, i copied the one below